full/src/main/java/apoc/uuid/UUIDNewProcedures.java

```diff
@@ -0,0 +1,117 @@
+package apoc.uuid;
+
+import apoc.Extended;
+import apoc.util.SystemDbUtil;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.kernel.api.procedure.SystemProcedure;
+import org.neo4j.procedure.*;
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static apoc.ApocConfig.apocConfig;
+import static apoc.uuid.UuidHandler.APOC_UUID_REFRESH;
+
+@Extended
+public class UUIDNewProcedures {
+    public static final String UUID_NOT_SET = APOC_UUID_REFRESH + " is not set. Please please set it in your apoc.conf";
```

Comment:
shouldn't it default to 60s ?

Reply:
By default I put `null` not to cause breaking-change with deprecated ones,
since they don't have a refresh handling: https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/3484/files#diff-24843e788bdf04d9e84495327581832cf31ff985f006cadbe92197ee9a706c12R78.